### PR TITLE
[CI:BUILD] Packit/rpm: fix aardvark-dns handling

### PR DESCRIPTION
**Pull request by lsm5**
Bodhi tasks are disabled in Packit config to allow manual submission of a single bodhi update containing netavark and aardvark-dns X.Y builds.

The netavark rpm will now depend on the minimum aardvark-dns X.Y build available to also account for patch releases. Ref: https://github.com/containers/netavark/issues/907#issuecomment-1914504620

Resolves: #907

### CODE REVIEW

**Comment by lsm5** [comment]
i'll try to check if this is handled correctly in copr as well before we merge.

**Comment by lsm5** [comment]
The rpm generated in the packit builds listed in CI can be installed with the current aardvark-dns rpm in `rhcontainerbot/podman-next`. So, AFAICT this should work.

This can also be partly verified with: `$ rpmspec -q --qf '%{Requires}\n' rpm/netavark.spec`. You'll need to update `Version` to `1.10.1` before you run that command.

**Comment by lsm5** [comment]
@mheon @Luap99 PTAL

**Review by Luap99 — APPROVED** [review]
/lgtm
/hold

**Comment by mheon** [comment]
LGTM
/hold cancel

**Comment by lsm5** [comment]
@Luap99 for whatever reason the lgtm label from your comment didn't get applied.

**Comment by lsm5** [comment]
manually adding lgtm label

**Pull request merged**
